app/views/events/index.html.erb

```diff
@@ -0,0 +1,53 @@
+<%= render 'layouts/navigation', current_user: @current_user%>
+
+<br><br>
+
+<table>
+    <thead>
+        <strong>PREVIOUS EVENTS</strong>
+    </thead>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <th>Location</th>
+            <th colspan="3">Date</th>
+            <th colspan="3">Creator</th>
+        </tr>
+
+        <% @events.past.each do |event| %>
+            <tr>
+                <td><%= event.description %></td>
+                <td><%= event.location %></td>
+                <td colspan="3"><%= event.date %></td>
+                <td colspan="3"><%= event.creator.name %></td>
+            </tr>
+        <% end %>
+    </tbody>
+</table>
+
+<br>
+
+<table>
+    <thead>
+        <strong>UPCOMING EVENTS</strong>
+    </thead>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <th>Location</th>
+            <th colspan="3">Date</th>
+            <th colspan="3">Creator</th>
+        </tr>
+
+        <% @events.upcoming.each do |event| %>
+            <tr>
+                <td><%= event.description %></td>
+                <td><%= event.location %></td>
+                <td colspan="3"><%= event.date %></td>
+                <td colspan="3"><%= event.creator.name %></td>
+            </tr>
+        <% end %>
```

Comment:
- [ ] Let's remove the logic from views.
  - Having block logic, if/else loops and loops, directly written in views makes you app slower since Rails will have to parse all that ruby in the erb format to get the values it needs for the actual html pages.
  - Instead use partials and helper methods.